ipaclient/remote_plugins/__init__.py

```diff
@@ -13,18 +13,16 @@
 from . import compat
 from . import schema
 from ipaclient.plugins.rpcclient import rpcclient
-from ipalib.constants import USER_CACHE_PATH
 from ipapython.dnsutil import DNSName
 
 logger = logging.getLogger(__name__)
 
 
 class ServerInfo(MutableMapping):
-    _DIR = os.path.join(USER_CACHE_PATH, 'ipa', 'servers')
-
     def __init__(self, api):
+        self._dir = os.path.join(api.env.cache_dir, "servers")
         hostname = DNSName(api.env.server).ToASCII()
-        self._path = os.path.join(self._DIR, hostname)
+        self._filename = os.path.join(self._dir, hostname)
         self._force_check = api.env.force_schema_check
```

Comment:
Do we think any users are relying on this class? Should we first deprecate the old values and maintain both?

Reply:
Neither GitHub search nor Google nor DuckDuckGo have a hit on `"ServerInfo._DIR"`. I could add a property for `_path` or change the code to use `_path` again, if you insist.

Besides the underscore prefix indicates that the attributes are an internal API detail.

Reply:
Works for me, thanks for looking.